2022-Verhagen-001/NASSA.yml

```diff
@@ -0,0 +1,38 @@
+id: 2022-Verhagen-001
+nassaVersion: 0.3.0
+moduleType: Algorithm
+title: Fertility_Rate.nlogo
+moduleVersion: 1.0.0
+contributors:
+ - name: Verhagen, Philip (with no accent marks, first upper-case letters, and surname matching the id and root directory name)
+   roles: [ "Author", "Compiler", "Contributor", "Copyright Holder", "Creator", "Thesis Advisor", "Translator" ]
```

Comment:
please select only the relevant roles and delete the others

Reply:
done